.eslintrc.json

```diff
@@ -7,6 +7,7 @@
     "mocha": true
   },
   "rules": {
+    "no-restricted-globals": ["error", "event"],
```

Comment:
why add this rule?

Reply:
I wanted to use `isNaN()` function, and it was giving an eslint error, so I tried to look for other solutions, and finally, I decided to add this rule.